.github/workflows/ghcr-actions.yml

```diff
@@ -0,0 +1,42 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: ghcr.io
+
+on:
+  push:
+  schedule:
+    - cron: "21 21 * * *"
+
+jobs:
+  clean-ghcr:
+    name: Delete old unused container images
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+    steps:
+      - name: Delete 'PR' containers older than a week
+        uses: snok/container-retention-policy@v2.0.0
+        with:
+          image-names: sensitive-data-archive
+          filter-tags: PR*,sha-*
+          cut-off: A week ago UTC
+          account-type: org
+          org-name: ${{ github.repository_owner }}
+          keep-at-least: 1
+          token: ${{ secrets.GHCR_TOKEN }}
+          timestamp-to-use: updated_at
+
+      - name: Delete all containers older than treee months, using a wildcard
+        uses: snok/container-retention-policy@v2.0.0
+        with:
+          image-names: sensitive-data-archive
+          filter-tags: v*
+          cut-off: Three months ago UTC
```

Comment:
Aren't these "released" versions, do we really want to clean those?

Reply:
Given that we get dependabot update every week, images that are more than 3 months old can be considered obsolete.

Reply:
I haven't kept track, how many of those are security updates?

My understanding is that most installations currently upgrade the pipeline more seldom than every three months. So, if these are the "released" artifact, I'm afraid we may run into breakages because of harsh cleaning (e.g. a pod failing, being rescheduled on another node that does not have image available and can't pull it because it's no longer available).

I also think we should keep "released" versions around for historical reason (e.g. being able to see the actual image used at a certain point in time) and the resource requirement for keeping those around should be very, very low.

(If these aren't the primary produced artifact, this does not apply.)

Reply:
If we see that we have to many images around we might prune the patch version related to older minor tags and only keep the major and minor versions.

Reply:
No objection.